Schedule.php

```diff
@@ -17,11 +20,29 @@ class Schedule extends Component
      */
     protected $_events = [];
 
+    /**
+     * The mutex implementation.
+     *
+     * @var \yii\mutex\Mutex
+     */
+    protected $_mutex;
+
     /**
      * @var string The name of cli script
      */
     public $cliScriptName = 'yii';
 
+    /**
+     * Schedule constructor.
+     * @param array $config
+     */
+    public function __construct(array $config = [])
+    {
+        $this->_mutex = Yii::$app->has('mutex') ? Yii::$app->get('mutex') : (new FileMutex());
```

Comment:
@VeerUP I think that is needed to add some notes to README about mutex configuration with link to [component documentation](http://www.yiiframework.com/doc-2.0/yii-mutex-mutex.html)